generators/lua/math.js

```diff
@@ -66,9 +66,14 @@ Blockly.Lua['math_single'] = function(block) {
         Blockly.Lua.ORDER_UNARY) || '0';
     return ['-' + arg, Blockly.Lua.ORDER_UNARY];
   }
-  if (operator == 'SIN' || operator == 'COS' || operator == 'TAN') {
+  if (operator == 'POW10') {
     arg = Blockly.Lua.valueToCode(block, 'NUM',
-        Blockly.Lua.ORDER_MULTIPLICATIVE) || '0';
```

Comment:
Because whoever created the Lua generator did a straight port from JavaScript.  In JS we generate:

`code = 'Math.sin(' + arg + ' / 180 * Math.PI)';`

Whereas in Lua we generate:

`code = 'math.sin(math.rad(' + arg + '))';`

Thus Lua was being asked to insert some unnecessary parens in some cases.